onnx/defs/gen_doc.py

```diff
@@ -81,32 +81,48 @@ def main(args):
         # inputs
         s += '\n#### Inputs'
         if schema.min_input != schema.max_input:
-            s += ' ({} - {})'.format(display_number(schema.min_input),
-                                   display_number(schema.max_input))
-        s += '\n\n'
-        if schema.input_desc:
-            s += '<dl>\n'
-            for idx, (input_name, input_desc) in enumerate(schema.input_desc):
-                s += '<dt><tt>{}</tt>{}</dt>\n'.format(
-                    input_name,
-                    ' (optional)' if idx in schema.optional_inputs else '')
-                s += '<dd>{}</dd>\n'.format(input_desc)
-            s += '</dl>\n'
+            s += '{} - {}'.format(display_number(schema.min_input),
+                                  display_number(schema.max_input))
+        s += '\n'
+        if schema.inputs:
+            s += '    <dl>\n'
```

Comment:
The generated documentation doesn't look right. Is it because of the extra indentation? I don't know markdown, but some change might be required.

Reply:
Format fixed now. It's more readable now. :). Thanks!